docs/modules/operation/pages/provisioning/hardware-inventory.adoc

```diff
@@ -0,0 +1,113 @@
+    <mibObj oid=".1.3.6.1.4.1.9.9.92.1.1.1.9"  type="string" alias="ceAssetSoftwareID" />
+    <mibObj oid=".1.3.6.1.4.1.9.9.92.1.1.1.10" type="string" alias="ceAssetSoftwareRevision" replace="entPhysicalSoftwareRev" />
+    <mibObj oid=".1.3.6.1.4.1.9.9.92.1.1.1.11" type="string" alias="ceAssetCLEI" />
+    <mibObj oid=".1.3.6.1.4.1.9.9.92.1.1.1.12" type="string" alias="ceAssetAlias" />
+</hw-extension>
+----
+
+In this example, only the nodes that match the `sysOidMask .1.3.6.1.4.1.9` (Cisco devices) are candidates to discover the custom `mibObj` entities defined on the extension group.
+The definition of `ceAssetSerialNumber` has been configured to replace one of the base attributes `entPhysicalSerialNum` so the adapter will know to override the generic Entity-MIB attribute instead of creating a new custom attribute entry.
+
+Once you have configured your `snmp-hardware-inventory-adapter-configuration.xml`, the Provisiond service attempts to collect hardware inventory data when it refreshes requisitions.
+After synchronizing the requisition, you should be able to see a `uei.opennms.org/internal/discovery/hardwareInventorySuccessful` event to confirm inventory was collected.
+This event is triggered only when the provisioning adapter detects changed hardware inventory for a node.
+If there is a problem retrieving or storing the inventory data, a `uei.opennms.org/internal/discovery/hardwareInventoryFailed` event will be sent instead.
+
+== Viewing collected hardware inventory information
+
+To view inventory information collected for a node, click the "Hardware Info" link on the node's page in the Web UI.
+If the node has hardware inventory, {page-component-title} displays all collected fields.
+
+TIP: It is also possible to retrieve and update hardware inventory information via the xref:development:rest/nodes.adoc#nodes-rest[Nodes REST endpoint].
+See the REST documentation for more information.
+
+== Using hardware information in events and notifications
+
+A set of placeholders are available in {page-component-title} to enrich the content of the events, alarms, and notifications.
+Because of the hierarchical nature of the hardware entities, you must provide two fields to obtain the value of the required attribute.
+These two fields are either the `entPhysicalIndex` or the `entPhysicalName` and the name of the attribute to use.
+
+The `entPhysicalIndex` guarantees that you always get the same entity.
+Using `entPhysicalName` is not always a unique way to identify an entity, depending on the device manufacturer.
+When using the name, the first entity that matches the given `entPhysicalName` is used.
+
+Specify a regular expression to validate against the `entPhysicalName` by prefixing the field with a `~`.
+When using regular expression matching, the first attribute matched is used.
+
+Sample placeholders naming structure:
+
+* %hardware[\{entPhysicalIndex}:\{attributeName}]%
```

Comment:
Do we need to keep the escape character in these three lines?

Reply:
Yes we do.  Otherwise there are Antora build errors.